zingolib/src/wallet/send.rs

```diff
@@ -274,7 +274,7 @@ impl LightWallet {
         let witness_trees = txmds_readlock
             .witness_trees
             .as_ref()
-            .expect("If we have spend capability we have trees");
+            .ok_or("No spend capability.")?;
```

Comment:
I think `ok_or_else` is preffered as its lazily evaluated
```suggestion
            .ok_or_else(|| "No spend capability.".to_string())?;
```